styles/globals.css

```diff
@@ -11,18 +11,25 @@ html {
   --dark-background-color: #20272f;
   --light-text-color: #ffffff;
   --dark-text-color: #20272f;
+  --dark-new-color: #022b3a;
```

Comment:
Instead of adding a new variable, you can change `dark-background-color` and `dark-text-color` to the new value